lib/galaxy/config/sample/job_conf.xml.sample_advanced

```diff
@@ -604,6 +604,12 @@
         </destination>
         <destination id="singularity_local" runner="local">
           <param id="singularity_enabled">true</param>
+          <!--Galaxy requests an isolated /tmp directory from singularity, which means
+              it will be mounted as read-only. This can cause some problems with tools that
+              do not use the TMP, TEMP, TMPDIR variable family properly. Setting
```

Comment:
> tools that do not use the TMP, TEMP, TMPDIR variable family properly

What does this mean?

Reply:
Proper applications create a temporary directory/file in the directory designated by TMPDIR. Shitty applications just hardcode `/tmp`. Which causes problems when `/tmp` is mounted read-only if `--no-mount tmp` is chosen and no other directory is mounted to the `/tmp` path.